ouroboros-network/src/Ouroboros/Network/Diffusion.hs

```diff
@@ -319,9 +327,10 @@ runDataDiffusion tracers
             sd
             (daLocalResponderApplication applications)
             localErrorPolicy
+          return ()
```

Comment:
Do we need this `return`? There's `void` applied to previous monadic action.

Reply:
No, the `return` isn't needed. Will remove it.